src/views/InvolvementsAll/index.js

```diff
@@ -83,12 +83,9 @@ const InvolvementsAll = ({ location, history }) => {
       setAllInvolvements(await involvementService.getAll(selectedSession));
       setTypes(await involvementService.getTypes(selectedSession));
       if (profile) {
-        setMyInvolvements(
-          await membershipService.getSessionMembershipsWithoutGuests(
-            profile.AD_Username,
-            selectedSession,
-          ),
-        );
+        membershipService
+          .getSessionMembershipsWithoutGuests(profile.AD_Username, selectedSession)
+          .then(setMyInvolvements);
```

Comment:
Couple of thoughts:
1. It's weird to change this syntax in this file without changing anything else in this file.
2. It's weird to change only this instance of this syntax in this file, when the exact same syntax is used directly above it
3. This actually breaks the loading behavior of this page. By using `.then` instead of `await`, we allow execution of the rest of this function before the promise is fulfilled, which means we set loading to false before `MyInvolvements` has loaded. That makes this change a bug.

Reply:
Yea, I think I reverted other changes here but somehow missed this one. Good catch with the bug.